drivers/hmis/app/graphql/types/hmis_schema/query_type.rb

```diff
@@ -396,6 +396,15 @@ def form_definition(id:)
       Hmis::Form::Definition.find(id)
     end
 
+    field :external_form_definition, Types::Forms::FormDefinition, null: true do
+      argument :identifier, String, required: true
+    end
+    def external_form_definition(identifier:)
+      raise 'Access denied' unless current_user.can_manage_external_form_submissions?
+
+      Hmis::Form::Definition.find_by(role: 'EXTERNAL_FORM', identifier: identifier)
```

Comment:
This won't be unique because there can be multiple forms with different version. Identifier & version are unique together (per the validation on the model), so this should be enough: `Hmis::Form::Definition.where(role: 'EXTERNAL_FORM', identifier: identifier).order(version: :desc).first`
```

It looks like we don't have a uniqueness constraint in the database, which we probably should.

Reply:
Great - thank you 👍